CondCore/JetMETPlugins/plugins/JetResolution_PayloadInspector.cc

```diff
@@ -349,6 +508,38 @@ namespace JME {
         sf_eta_norm->Draw("][ same");
         sf_eta_norm->Draw("axis same");
 
+        leg_eta->AddEntry(sf_eta_norm, (tag_ver + '_' + tag_jet).c_str(), "l");
+        leg_eta->AddEntry((TObject*)nullptr, "JetPt=500", "");
+        leg_eta->Draw();
+
+        if (is_2bin == true) {
+          canvas.cd(2);
+          sf_pt_up->SetTitle("ScaleFactor vs. p_{T}");
+          sf_pt_up->SetXTitle("p_{T} [GeV]");
+          sf_pt_up->SetYTitle("Scale Factor");
+          sf_pt_up->SetLineStyle(7);
+          sf_pt_up->SetLineWidth(3);
+          sf_pt_up->SetFillColorAlpha(kGray, 0.5);
+          sf_pt_up->SetMaximum(sf_pt_up->GetMaximum() * 1.25);
+          sf_pt_up->SetMinimum(0.);
+          sf_pt_up->Draw("][");
+
+          sf_pt_down->SetLineStyle(7);
+          sf_pt_down->SetLineWidth(3);
+          sf_pt_down->SetFillColorAlpha(kWhite, 1);
+          sf_pt_down->Draw("][ same");
+
+          sf_pt_norm->SetLineStyle(1);
+          sf_pt_norm->SetLineWidth(5);
+          sf_pt_norm->SetFillColor(0);
+          sf_pt_norm->Draw("][ same");
+          sf_pt_norm->Draw("axis same");
+
+          leg_pt->AddEntry(sf_pt_norm, (tag_ver + '_' + tag_jet).c_str(), "l");
+          leg_pt->AddEntry((TObject*)nullptr, "JetEta=2.55", "");
```

Comment:
why is this hardcoded to 2.55? Also isnt this less than 2.55 instead of the equal sign?

Reply:
For the Jet resolution part, I follow the plots in the presentation of Andrea that rho and eta or pt are fixed. As to SF, for this two-bin case, I pick up the eta region that has most records. (here 2.5-2.6, 2.55 is the central value)

Reply:
Yes, the old JEC/JER web viewer shown in Andreas' slides uses fixed values and in the latest version users can choose from a drop-down menu, PI can only take float inputs. I need to check if some default values can be set, otherwise it's rather not-ideal. I'll deal with this after finish the JEC PI.